dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -73,15 +73,15 @@ ENV BEAT_SETUID_AS={{ .user }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do \
         microdnf update -y && \
-        microdnf install -y tar gzip findutils shadow-utils ca-certificates gawk libcap xz && \
+        microdnf install -y tar gzip findutils shadow-utils ca-certificates gawk libcap xz systemd && \
         microdnf clean all && \
         exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
 {{- end }}
 
 {{- if contains .from "wolfi" }}
 RUN for iter in {1..10}; do \
         apk fix && \
-        apk add --no-cache ca-certificates curl gawk shadow bash && \
+        apk add --no-cache ca-certificates curl gawk shadow bash systemd && \
```

Comment:
We shouldn't include systemd in wolfi because it is going to significantly broaden it's exposure to CVEs, the point of Wolfi is to minimize the OS level CVEs we are exposed to. Same rationale as Filebeat: https://github.com/elastic/beats/pull/44056/files#r2064612269